src/testing/slogtest/slogtest.go

```diff
@@ -33,7 +33,22 @@ type testCase struct {
 	checks []check
 }
 
+// PERSONAL DEV NOTES:
+// here's the main issue https://github.com/golang/go/issues/62280
+
 var cases = []testCase{
+	{
+		// some value types:KindInt64 , KindUint64, KindFloat64, KindBool, KindDuration, KindTime
+		name:        "KindInt64",
+		explanation: withSource("test case to log KindInt64"),
+		f: func(l *slog.Logger) {
+			l.Info("message", "int64key", "123124")
+		},
+		checks: []check{
```

Comment:
@jba  - hey, this is what i have now - just to be clear: we input a string and we expect slog to figure out the type here? or is it the otherway around ?

Reply:
Actually my comment "slogtest deliberately uses only one or two value kinds so users don't have to write full parsers. I don't want to change that" applies to the case of KindInt64 and so on.

The only ones that I think we should do are discussed in https://github.com/golang/go/issues/62280#issuecomment-1710530497.
I would start with "a Record with an empty PC".

Reply:
what is a `PC` again @jba  ?